src/hb-ot-stat-table.hh

```diff
@@ -57,6 +57,19 @@ enum
   // Reserved = 0xFFFC				/* Reserved for future use — set to zero. */
 };
 
+static bool axis_value_is_outside_axis_range (hb_tag_t axis_tag, float axis_value,
+                                              const hb_hashmap_t<hb_tag_t, Triple> *user_axes_location)
+{
+  if (!user_axes_location->has (axis_tag))
+    return false;
+
+  Triple axis_range = user_axes_location->get (axis_tag);
+  if (axis_value < axis_range.minimum || axis_value > axis_range.maximum)
```

Comment:
You can simplify this to:
return (axis_value < axis_range.minimum || axis_value > axis_range.maximum)